docs/features/scenario-workflow/brief.md

```diff
@@ -0,0 +1,110 @@
+* [Interactive prototype from Project Dashboard (before setting parameters and Run)](https://www.figma.com/proto/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?page-id=6694%3A10654&node-id=6727%3A12685&viewport=333%2C48%2C0.53&scaling=min-zoom&starting-point-node-id=6727%3A12685&show-proto-sidebar=1)
+* [Interactive prototype of editing](https://www.figma.com/proto/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?page-id=6694%3A10654&node-id=6837%3A18442&viewport=333%2C48%2C0.53&scaling=min-zoom&starting-point-node-id=6837%3A18442&show-proto-sidebar=1)
+
+## Invalidating information
+
+(work in progress, from https://docs.google.com/document/d/18lIqOVrn5QXE4QuAaqrTLndQa0-q8tmkyrQJfGlLu-8/edit)
+
+### When a user makes any changes that require any kind of recalculation(s) before running Marxan again
+
+E.g: A change on the Protected Areas section will require recalculations of the
+features before re-running Marxan.
+
+* Interface shows:
+  * Red dots in all the tabs outdated that needs to be recomputed.
+  * A disclaimer + button saying: “Your information and solutions are outdated” <Recalculate>
+* When user clicks recalculate, the system runs all the needed calculations.
+  System should detect all the calculations needed, and run them)
+  * User could directly re-run Marxan, in this case, the system will re-do all the calculations needed. 
+* When calculations are done:
+  * all the red dots disappear but the one on solutions.
+  * generic disclaimer + button disappear
+* User can make more changes or go to “solutions” tab
+* When finally on solution tab, user clicks re-run Marxan
+* System run Marxan
+* All disclaimers and red dots disappear.
+
+### When a user makes a change that while causing some information to be outdated, does not require recalculations
+
+Eg: a change on the Protected Areas section will invalidate the BLM calibration
+but is not really needed to re-run that calibration, just letting the user know
+that part is outdated.
+
+* Interface shows:
+  * Orange dots in all the tabs that contain outdated information.
+  * Inside each tab: A warning explaining what is going on. 
+* When the user updates the information (for example by running a BLM
+  calibration afresh), warnings disappear. 
+
+## Application support
+
+In order to reliably support this scenario editing workflow:
+
+* the API should keep track of which parts of the scenario are up to date and
+  which are not
+* it should provide this information to the frontend app, on request, so that
+  appropriate warnings and calls to action may be displayed
+* it should prevent actions from being carried out on stale data
+* the frontend app should display data staleness warnings with relevant
+  importance color-coding (see previous section) and calls to action:
+  * required: recalculate scenario, re-run Marxan
+  * informational: refresh gap analysis, re-run BLM calibration
```

Comment:
@anamontiaga would you mind suggesting anything else that the app will depend on?